libseccomp/src/error.rs

```diff
@@ -157,9 +154,46 @@ impl SeccompError {
         }
     }
 
+    /// Query the system's raw error code returned when something goes wrong
+    /// in the libc and the kernel.
+    ///
+    /// This function will be useful for users who want to extract the system's
+    /// error code directly returned by [`ScmpFilterAttr::ApiSysRawRc`](`crate::ScmpFilterAttr::ApiSysRawRc`).
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new(ScmpAction::Allow)?;
+    /// ctx.set_api_sysrawrc(true)?;
+    /// match ctx.export_pfc(&mut -1) {
+    ///     Err(e) => {
+    ///         eprintln!("Error: {e}");
+    ///         if let Some(sys) = e.sysrawrc() {
+    ///             eprintln!("The system's raw error code: {sys}");
+    ///             assert_eq!(sys, -libc::EBADF);
+    ///         } else {
+    ///             eprintln!("The libseccomp errno: {}", e.errno().unwrap());
+    ///         }
+    ///     }
+    ///     _ => println!("No error"),
+    /// }
+    /// # Ok::<(), Box<dyn std::error::Error>>(())
+    /// ```
+    pub fn sysrawrc(&self) -> Option<i32> {
+        if let ErrorKind::SysRawRc(errno) = self.kind {
+            Some(errno)
+        } else {
+            None
```

Comment:
That's good idea. Added `to_sysrawrc()` for `SeccompErrno` as an internal api and modified the description and the example code. The `SeccompError::sysrawrc()` function will be useful for users who want to convert the errno returned by the libseccomp API to a negative integer (`-libc::E***`)

```rust
/// This function will be useful for users who want to extract the system's                                 
/// error code directly returned by [`ScmpFilterAttr::ApiSysRawRc`](`crate::ScmpFilterAttr::ApiSysRawRc`)   
/// , or get the errno returned by the libseccomp API as a negative integer rather than [`SeccompErrno`].   
```